syntax = "proto3";

package predictions;

option go_package = "path/to/your/package/predictions";

service PredictionService {
    rpc Predict(PredictionRequest) returns (PredictionResponse);
}

message PredictionRequest {
    string team1 = 1;
    string team2 = 2;
    int32 year = 3;
}

message PredictionResponse {
    float prediction = 1;
}